classes/Organisations.js

```diff
@@ -38,17 +37,24 @@ module.exports = (tenant /* : Tenant */) =>
           new TypeError('Must supply "assetFileName" as a string'),
         )
       }
-      if (typeof organisationId !== 'string') {
-        return Promise.reject(
-          new TypeError('Must supply "organisationId" as a string'),
-        )
-      }
 
       return super
-        .postRequest('/asset-upload-credentials', {
-          assetPath: `assets/${assetFileName}`,
-          organisationId,
+        .getRequest('/organisations')
+        .then((searchResponse) => {
+          if (!searchResponse.organisations) {
```

Comment:
```suggestion
          if (!searchResponse.organisations || !searchResponse.organisations[0]) {
```

should also cater for an empty array.